packages/shopify-app-session-storage-test-utils/src/battery-of-tests.ts

```diff
@@ -137,25 +138,25 @@ export function batteryOfTests(storageFactory: () => Promise<SessionStorage>) {
         id: `${prefix}_1`,
         shop: 'find-shop1-sessions.myshopify.io',
         state: 'state',
-        isOnline: true,
```

Comment:
Many of our tests were set up with invalid data. The current flow of code cannot create session data that looks like this. The new update to the session class in the shopify-api-js can no longer handle this invalid data setup.

Reply:
Agreed, these just seem like invalid test cases.